### DISCUSSION TITLE
ng lint --fix

### DISCUSSION THREAD

**Discussion by aleksa520** [angular-eslint/angular-eslint#1198] [Q&A] [answered]
I can't find anything about **ng lint --fix** command in [documentation](https://angular.io/cli/lint). Is --fix command supported?
Is there any workaround for this?

**Comment by JamesHenry** [comment, marked as answer]
We do not control the official Angular documentation, that is maintained by the Angular team.

Yes the `fix` option is supported, as with all other `ng` commands you could run `--help` to see this and all other available options, i.e. `ng lint --help`.

Ultimately that information is being pulled from the `schema.json` of the `@angular-eslint/builder` package: https://github.com/angular-eslint/angular-eslint/blob/main/packages/builder/src/schema.json
